File: src/Generated20260221.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260221 {
public:
    // Generated class 20260221_120649_296999
    int value = 816;
    string created = "20260221_120649_296999";
    
    int process() {
        int result = value;
        for (int i = 0; i < 5; i++) {
            result += i * 8;
        }
        return result;
    }
};

int main() {
    Generated20260221 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260221.go
package main

import "fmt"

type Generated20260221 struct {
    // Generated struct 20260221_142941_088821
    Value int
    Created string
}

func (c *Generated20260221) Process() int {
    result := c.Value
    for i := 0; i < 10; i++ {
        result += i * 4
    }
    return result
}

func main() {
    obj := Generated20260221{Value: 292, Created: "20260221_142941_088821"}
    fmt.Println(obj.Process())
}